images/build/Dockerfiles/ltsVersions.Dockerfile

```diff
@@ -171,6 +174,8 @@ RUN set -ex \
         libonig-dev \
     && rm -rf /var/lib/apt/lists/* \
     && cp -f $imagesDir/build/benv.sh /opt/oryx/benv \
+    && cp -f $imagesDir/installPlatform.sh /opt/oryx/installPlatform \
```

Comment:
So installPlatform will act like a command same as benv in the container? Are we able to run command like 'installPlatform -p python -v 3.9' inside the image?

Reply:
`installPlatform` will be used to install platform versions either when building the image in our CI or even when dynamically installing platforms during customer app's build time too. Once dynamic install is approved for runtime images, we can use this script to install dynamically in runtime images too